src/microbe_stage/systems/EngulfingSystem.cs

```diff
@@ -186,6 +186,17 @@ protected override void Update(float delta, in Entity entity)
             return;
         }
 
+        // Not full anymore tutorial (end trigger for engulfment full tutorial)
+        if (engulfer.UsedEngulfingCapacity <= engulfer.EngulfingSize * 0.5f)
```

Comment:
This is not the most important thing in the world, so for ease of reading this code file I'd say this block of code should instead be on around line 259 (after the sound, but before colony handling).